src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -95,7 +99,7 @@ export const ProgressIndicator: ChakraComponent<
       if (indicatorType === "circular") {
         // For the small size, since the label won't be visible, we need to add
```

Comment:
I don't think this comment is true anymore since the circular small size variant does display the label on the right side.